StationeersLaunchPad/LaunchPadConfig.cs

```diff
@@ -371,7 +380,33 @@ private static void Initialize()
       Mods.Add(new ModInfo { Source = ModSource.Core });
     }
 
-    private static void LoadConfig()
+    private static bool CheckIgnoreOtherFile()
+    {
+      try
+      {
+        var ignoreFilePath = Path.Combine(Paths.GameRootPath, "mods", "ignoreOther.txt");
```

Comment:
I think this is probably better as a config field instead of using a marker file. The config file is stored in the `Bepinex/config` directory in the game folder so it will come along with a copied game install.